rialto_airflow/harvest/pubmed.py

```diff
@@ -104,6 +148,16 @@ def pmids_from_orcid(orcid):
     return _pubmed_search_api(f"{orcid}[auid]")
 
 
+def pmids_from_dois(dois: list[str]) -> list[str]:
+    """
+    Returns PMIDs associated with given list of DOIs.
+    """
+
+    query = " OR ".join([f'"{s}"' for s in dois])
+
+    return _pubmed_search_api(f"{query}")
```

Comment:
query Pubmed API for PMIDs given DOIs (it's basically a term search quoting the DOIs and ORing them together